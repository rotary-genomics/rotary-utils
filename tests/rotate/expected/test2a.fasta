>sequence1
AAAACCCCCTTTTTGGGGGA
>sequence2 # comment
TTCCCAAAGGGT
>sequence3
GAATTCCG
