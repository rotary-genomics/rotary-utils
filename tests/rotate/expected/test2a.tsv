sequence_id	sequence_length_bp	rotate_type	rotate_value
sequence1	20	position	1
sequence2	12	position	1
sequence3	8	position	1
